clients/java-logger/grpc-test-app/src/main/java/abcxyz/lumberjack/test/talker/TalkerService.java

```diff
@@ -111,17 +114,22 @@ private void blockUntilShutdown() throws InterruptedException {
     }
   }
 
+  public static String parsePublicKey() {
```

Comment:
It might make more sense to put this method inside JWKHandler as it is only used there. also private instead of public.